### ISSUE TITLE
[BITV] 9.1.3.2/4.1 - When the image viewer is open, in some cases, not only the alternative text of the visible image is read aloud but also that of a previous one. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41840]
Additionally, in some case, visually hidden elements of the video player are announced, such as sliders.

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-1782286125-v4-n1

**Comment by JuliaKirschenheuter** [comment]
@artonge could you please help us with this ticket?

**Comment by ShGKme** [comment]
I have a draft, but currently it doesn't work with external video player.
